src/utils/clear-mdast-positions.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const removePosition = require('unist-util-remove-position');
+const { numericLogLevel } = require('@adobe/helix-log');
+
+async function clearPositions({ content: { mdast = {} } }, { logger }) {
+  if (numericLogLevel(logger.level) < numericLogLevel('debug')) {
```

Comment:
It's generally (and in this case, too) a good practice to have the conditionals in the pipeline, and not in the step. This makes your step unit tests easier (no branching needed) and separates the conditional from the working code.